tests/foreman/rhai/test_rhai.py

```diff
@@ -130,6 +131,9 @@ def test_unregister_system_from_rhai(self):
                 session.nav.click(
                     locators['insights.unregister_button']
                 )
+                self.browser.refresh()
+                time.sleep(30)
```

Comment:
Have you tried using `self.wait_for_ajax()`? Using hard sleep is not a good approach and is used only in cases that there are not other options.